pkg/constants/constants.go

```diff
@@ -48,11 +48,20 @@ const MinIOVolumeMountPath = "/export"
 const MinIOVolumeSubPath = ""
 
 // DefaultMinIOImage specifies the default MinIO Docker hub image
-const DefaultMinIOImage = "minio/minio:RELEASE.2020-04-23T00-58-49Z"
+const DefaultMinIOImage = "minio/minio:RELEASE.2020-05-01T22-19-14Z"
+
+// DefaultMCImage specifies the default mc Docker hub image
+const DefaultMCImage = "minio/mc:RELEASE.2020-04-25T00-43-23Z"
 
 // MinIOServerName specifies the default container name for MinIOInstance
 const MinIOServerName = "minio"
 
+// MirorContainerName specifies the default container name for MirrorInstance
+const MirorContainerName = "mirror"
+
+// MirrorJobRestartPolicy specifies the restart policy for the job created for mirroring
+const MirrorJobRestartPolicy = "OnFailure"
```

Comment:
We have constants defined for RestartPolicy in corev1 pkg. Can we use that instead?

https://godoc.org/k8s.io/api/core/v1#RestartPolicy

Reply:
good catch. fixed @Praveenrajmani